diff --git a/Functions/dbo.ufn_parsefind.sql b/Functions/dbo.ufn_parsefind.sql
new file mode 100644
index 0000000..4a2c631
--- /dev/null
+++ b/Functions/dbo.ufn_parsefind.sql
@@ -0,0 +1,76 @@
+SET QUOTED_IDENTIFIER ON
+GO
+SET ANSI_NULLS ON
+GO
+
+
+
+/*
+
+Copyright © 2009 - John Burnette -- All Rights Reserved
+--Modified 04/18/2016 - Travis Parker (return NULL when fewer occurences of delimiter are found than passed in)
+*/
+
+CREATE FUNCTION [dbo].[ufn_parsefind]
+(
+     @EntString  VARCHAR(MAX),
+     @Delimiter  VARCHAR(10),
+     @Occurrence BIGINT
+)
+RETURNS VARCHAR(MAX)
+AS
+     BEGIN
+
+         DECLARE @CurString VARCHAR(MAX);
+         DECLARE @Pos BIGINT;
+         DECLARE @Loop BIGINT;
+	    DECLARE @DelOrig VARCHAR(10); --TP
+
+         -- REQUIRE DELIMITER AT END OF STRING
+
+         IF RIGHT(@EntString, 1) <> @Delimiter
+             SET @EntString = @EntString + @Delimiter;
+
+         -- ESTABLISH CORRECT SYNTAX FOR DELIMITER IN PATINDEX FUNCTION
+	    SET @DelOrig = @Delimiter --TP
+         SET @Delimiter = '%'+@Delimiter+'%';
+
+         SET @Loop = 1;
+         SET @Pos = PATINDEX(@Delimiter, @EntString);
+
+	    SET @Pos = (SELECT CASE WHEN LEN(@EntString) - LEN(REPLACE(@EntString,@DelOrig,'')) < @Occurrence THEN 0 ELSE @Pos END ) --TP
+
+         -- LOOP THROUGH IF DELIMTERS FOUND
+
+         IF @Pos = 0
+             BEGIN
+                 SET @CurString = NULL;
+             END;
+         ELSE
+             BEGIN
+                 WHILE @Loop <= @Occurrence
+                       AND @Pos <> 0
+                     BEGIN
+                         SET @Pos = PATINDEX(@Delimiter, @EntString);
+                         SET @CurString = LEFT(@EntString, @Pos - 1);
+                         SET @EntString = RIGHT(@EntString, LEN(@EntString) - LEN(@CurString) - 1);
+                         SET @Loop = @Loop + 1;
+                     END;
+             END;
+
+         -- DEFAULT A NULL FOR BLANK VALUES
+
+         IF ISNULL(@CurString, '') = ''
+            OR LEN(@CurString) < 1
+             SET @CurString = NULL;
+    
+         -- RETURN VALUE
+
+         RETURN @CurString;
+     END;
+
+
+
+
+
+GO
diff --git a/RedGateDatabaseInfo.xml b/RedGateDatabaseInfo.xml
new file mode 100644
index 0000000..d1e3a18
--- /dev/null
+++ b/RedGateDatabaseInfo.xml
@@ -0,0 +1,62 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DatabaseInformation Version="2">
+  <ScriptFileEncoding>UTF8</ScriptFileEncoding>
+  <DefaultCollation>SQL_Latin1_General_CP1_CI_AS</DefaultCollation>
+  <DefaultSchema>dbo</DefaultSchema>
+  <DefaultUser>dbo</DefaultUser>
+  <DefaultFilegroup>PRIMARY</DefaultFilegroup>
+  <DatabaseVersion>12</DatabaseVersion>
+  <IsAzure>False</IsAzure>
+  <MaxDataFileSize>10485760</MaxDataFileSize>
+  <WriteToFileOptions>
+    <Prefixes>
+      <None>
+      </None>
+      <Table>Tables</Table>
+      <StoredProcedure>Stored Procedures</StoredProcedure>
+      <View>Views</View>
+      <Default>Defaults</Default>
+      <FullTextCatalog>Storage\Full Text Catalogs</FullTextCatalog>
+      <Function>Functions</Function>
+      <Role>Security\Roles</Role>
+      <Rule>Rules</Rule>
+      <User>Security\Users</User>
+      <UserDefinedType>Types\User-defined Data Types</UserDefinedType>
+      <Trigger>
+      </Trigger>
+      <DdlTrigger>Database Triggers</DdlTrigger>
+      <Assembly>Assemblies</Assembly>
+      <Synonym>Synonyms</Synonym>
+      <XmlSchemaCollection>Types\XML Schema Collections</XmlSchemaCollection>
+      <MessageType>Service Broker\Message Types</MessageType>
+      <Contract>Service Broker\Contracts</Contract>
+      <Queue>Service Broker\Queues</Queue>
+      <Service>Service Broker\Services</Service>
+      <Route>Service Broker\Routes</Route>
+      <EventNotification>Service Broker\Event Notifications</EventNotification>
+      <PartitionScheme>Storage\Partition Schemes</PartitionScheme>
+      <PartitionFunction>Storage\Partition Functions</PartitionFunction>
+      <Field>
+      </Field>
+      <Index>
+      </Index>
+      <Schema>Security\Schemas</Schema>
+      <ServiceBinding>Service Broker\Remote Service Bindings</ServiceBinding>
+      <Certificate>Security\Certificates</Certificate>
+      <SymmetricKey>Security\Symmetric Keys</SymmetricKey>
+      <AsymmetricKey>Security\Asymmetric Keys</AsymmetricKey>
+      <CheckConstraint>
+      </CheckConstraint>
+      <FullTextStoplist>Storage\Full Text Stoplists</FullTextStoplist>
+      <ExtendedProperty>Extended Properties</ExtendedProperty>
+      <Data>Data</Data>
+      <Sequence>Sequences</Sequence>
+      <SearchPropertyList>Search Property Lists</SearchPropertyList>
+      <SecurityPolicy>Security Policies</SecurityPolicy>
+    </Prefixes>
+    <DataWriteAllFilesInOneDirectory>True</DataWriteAllFilesInOneDirectory>
+  </WriteToFileOptions>
+  <DataFileSet>
+    <Count>0</Count>
+  </DataFileSet>
+</DatabaseInformation>
\ No newline at end of file
diff --git a/Security/Roles/db_owner.sql b/Security/Roles/db_owner.sql
new file mode 100644
index 0000000..a4b5318
--- /dev/null
+++ b/Security/Roles/db_owner.sql
@@ -0,0 +1,4 @@
+EXEC sp_addrolemember N'db_owner', N'INTERNAL\CHSTESTDB02$'
+GO
+EXEC sp_addrolemember N'db_owner', N'INTERNAL\Paul.Johnson'
+GO
diff --git a/Security/Schemas/dw.sql b/Security/Schemas/dw.sql
new file mode 100644
index 0000000..7b7cc35
--- /dev/null
+++ b/Security/Schemas/dw.sql
@@ -0,0 +1,3 @@
+CREATE SCHEMA [dw]
+AUTHORIZATION [dbo]
+GO
diff --git a/Security/Users/INTERNAL_CHSTESTDB02$.sql b/Security/Users/INTERNAL_CHSTESTDB02$.sql
new file mode 100644
index 0000000..c4a6f0e
--- /dev/null
+++ b/Security/Users/INTERNAL_CHSTESTDB02$.sql
@@ -0,0 +1,6 @@
+IF NOT EXISTS (SELECT * FROM master.dbo.syslogins WHERE loginname = N'INTERNAL\CHSTESTDB02$')
+CREATE LOGIN [INTERNAL\CHSTESTDB02$] FROM WINDOWS
+GO
+CREATE USER [INTERNAL\CHSTESTDB02$] FOR LOGIN [INTERNAL\CHSTESTDB02$]
+GO
+GRANT CONNECT TO [INTERNAL\CHSTESTDB02$]
diff --git a/Security/Users/INTERNAL_Paul.Johnson.sql b/Security/Users/INTERNAL_Paul.Johnson.sql
new file mode 100644
index 0000000..d0ea22e
--- /dev/null
+++ b/Security/Users/INTERNAL_Paul.Johnson.sql
@@ -0,0 +1,6 @@
+IF NOT EXISTS (SELECT * FROM master.dbo.syslogins WHERE loginname = N'INTERNAL\Paul.Johnson')
+CREATE LOGIN [INTERNAL\Paul.Johnson] FROM WINDOWS
+GO
+CREATE USER [INTERNAL\Paul.Johnson] FOR LOGIN [INTERNAL\Paul.Johnson]
+GO
+GRANT CONNECT TO [INTERNAL\Paul.Johnson]
diff --git a/Stored Procedures/dbo.prDV_RAPS_LoadSats.sql b/Stored Procedures/dbo.prDV_RAPS_LoadSats.sql
new file mode 100644
index 0000000..fc17a11
--- /dev/null
+++ b/Stored Procedures/dbo.prDV_RAPS_LoadSats.sql
@@ -0,0 +1,469 @@
+SET QUOTED_IDENTIFIER ON
+GO
+SET ANSI_NULLS ON
+GO
+
+
+
+
+
+-- =============================================
+-- Author:		Travis Parker
+-- Create date:	06/10/2016
+-- Description:	Loads all the Satellites from the RAPS Response staging tables
+-- Usage:			
+--		  EXECUTE dbo.prDV_RAPS_LoadSats
+-- =============================================
+
+CREATE PROCEDURE [dbo].[prDV_RAPS_LoadSats]
+AS
+    BEGIN
+
+        SET NOCOUNT ON;
+
+        BEGIN TRY
+
+		  --LOAD RAPS AAA
+            INSERT  INTO dbo.S_RAPS_Response_AAA
+                    ( S_RAPS_Response_AAA_RK ,
+                      H_RAPS_Response_RK ,
+                      RecordID ,
+                      SubmitterID ,
+                      FileID ,
+                      TransactionDate ,
+                      ProdTestIND ,
+                      FileDiagType ,
+                      HashDiff ,
+                      LoadDate ,
+                      RecordSource 
+		          )
+                    SELECT  r.S_RAPS_Response_AAA_RK ,
+                            r.H_RAPS_Response_RK ,
+                            r.RecordID ,
+                            r.SubmitterID ,
+                            r.FileID ,
+                            r.TransactionDate ,
+                            r.ProdTestIND ,
+                            r.FileDiagType ,
+                            r.HashDiff ,
+                            r.LoadDate ,
+                            r.RecordSource
+                    FROM    CHSStaging.raps.vwRAPS_RESPONSE_AAA r
+                            LEFT JOIN dbo.S_RAPS_Response_AAA a ON a.H_RAPS_Response_RK = r.H_RAPS_Response_RK
+                                                              AND a.RecordEndDate IS NULL
+                                                              AND a.HashDiff = r.HashDiff
+                    WHERE   a.S_RAPS_Response_AAA_RK IS NULL; 
+		  
+		  --RECORD END DATE CLEANUP
+            UPDATE  dbo.S_RAPS_Response_AAA
+            SET     RecordEndDate = ( SELECT    DATEADD(ss, -1,
+                                                        MIN(z.LoadDate))
+                                      FROM      dbo.S_RAPS_Response_AAA AS z
+                                      WHERE     z.H_RAPS_Response_RK = a.H_RAPS_Response_RK
+                                                AND z.LoadDate > a.LoadDate
+                                    )
+            FROM    dbo.S_RAPS_Response_AAA a
+            WHERE   a.RecordEndDate IS NULL;
+		  
+
+		  --LOAD RAPS BBB
+            INSERT  INTO dbo.S_RAPS_Response_BBB
+                    ( S_RAPS_Response_BBB_RK ,
+                      H_RAPS_Response_RK ,
+                      RecordID ,
+                      SeqNo ,
+                      PlanNo ,
+                      OverpaymentID ,
+                      OverpaymentIDErrorCode ,
+                      PaymentYear ,
+                      PaymentYearErrorCode ,
+                      HashDiff ,
+                      LoadDate ,
+                      RecordSource 
+		          )
+                    SELECT  r.S_RAPS_Response_BBB_RK ,
+                            r.H_RAPS_Response_RK ,
+                            r.RecordID ,
+                            r.SeqNo ,
+                            r.PlanNo ,
+                            r.OverpaymentID ,
+                            r.OverpaymentIDErrorCode ,
+                            r.PaymentYear ,
+                            r.PaymentYearErrorCode ,
+                            r.HashDiff ,
+                            r.LoadDate ,
+                            r.RecordSource
+                    FROM    CHSStaging.raps.vwRAPS_RESPONSE_BBB r
+                            LEFT JOIN dbo.S_RAPS_Response_BBB b ON b.H_RAPS_Response_RK = r.H_RAPS_Response_RK
+                                                              AND b.RecordEndDate IS NULL
+                                                              AND b.HashDiff = r.HashDiff
+                    WHERE   b.S_RAPS_Response_BBB_RK IS NULL; 
+
+
+		  --RECORD END DATE CLEANUP
+            UPDATE  dbo.S_RAPS_Response_BBB
+            SET     RecordEndDate = ( SELECT    DATEADD(ss, -1,
+                                                        MIN(z.LoadDate))
+                                      FROM      dbo.S_RAPS_Response_BBB AS z
+                                      WHERE     z.H_RAPS_Response_RK = a.H_RAPS_Response_RK
+                                                AND z.LoadDate > a.LoadDate
+                                    )
+            FROM    dbo.S_RAPS_Response_BBB a
+            WHERE   a.RecordEndDate IS NULL;
+
+
+		  --LOAD RAPS CCC
+            INSERT  INTO dbo.S_RAPS_Response_CCC
+                    ( S_RAPS_Response_CCC_RK ,
+                      H_RAPS_Response_RK ,
+                      RecordID ,
+                      SeqNo ,
+                      SeqErrorCode ,
+                      PatientControlNo ,
+                      HicNo ,
+                      HicErrorCode ,
+                      PatientDOB ,
+                      DOBErrorCode ,
+                      ProviderType1 ,
+                      FromDate1 ,
+                      ThruDate1 ,
+                      DeleteIND1 ,
+                      DiagnosisCode1 ,
+                      DiagClstrErrorA1 ,
+                      DiagClstrErrorB1 ,
+                      ProviderType2 ,
+                      FromDate2 ,
+                      ThruDate2 ,
+                      DeleteIND2 ,
+                      DiagnosisCode2 ,
+                      DiagClstrErrorA2 ,
+                      DiagClstrErrorB2 ,
+                      ProviderType3 ,
+                      FromDate3 ,
+                      ThruDate3 ,
+                      DeleteIND3 ,
+                      DiagnosisCode3 ,
+                      DiagClstrErrorA3 ,
+                      DiagClstrErrorB3 ,
+                      ProviderType4 ,
+                      FromDate4 ,
+                      ThruDate4 ,
+                      DeleteIND4 ,
+                      DiagnosisCode4 ,
+                      DiagClstrErrorA4 ,
+                      DiagClstrErrorB4 ,
+                      ProviderType5 ,
+                      FromDate5 ,
+                      ThruDate5 ,
+                      DeleteIND5 ,
+                      DiagnosisCode5 ,
+                      DiagClstrErrorA5 ,
+                      DiagClstrErrorB5 ,
+                      ProviderType6 ,
+                      FromDate6 ,
+                      ThruDate6 ,
+                      DeleteIND6 ,
+                      DiagnosisCode6 ,
+                      DiagClstrErrorA6 ,
+                      DiagClstrErrorB6 ,
+                      ProviderType7 ,
+                      FromDate7 ,
+                      ThruDate7 ,
+                      DeleteIND7 ,
+                      DiagnosisCode7 ,
+                      DiagClstrErrorA7 ,
+                      DiagClstrErrorB7 ,
+                      ProviderType8 ,
+                      FromDate8 ,
+                      ThruDate8 ,
+                      DeleteIND8 ,
+                      DiagnosisCode8 ,
+                      DiagClstrErrorA8 ,
+                      DiagClstrErrorB8 ,
+                      ProviderType9 ,
+                      FromDate9 ,
+                      ThruDate9 ,
+                      DeleteIND9 ,
+                      DiagnosisCode9 ,
+                      DiagClstrErrorA9 ,
+                      DiagClstrErrorB9 ,
+                      ProviderType10 ,
+                      FromDate10 ,
+                      ThruDate10 ,
+                      DeleteIND10 ,
+                      DiagnosisCode10 ,
+                      DiagClstrErrorA10 ,
+                      DiagClstrErrorB10 ,
+                      CorrectedHicNo ,
+                      RiskAssessmentCode1 ,
+                      RiskAssessmentCodeError1 ,
+                      RiskAssessmentCode2 ,
+                      RiskAssessmentCodeError2 ,
+                      RiskAssessmentCode3 ,
+                      RiskAssessmentCodeError3 ,
+                      RiskAssessmentCode4 ,
+                      RiskAssessmentCodeError4 ,
+                      RiskAssessmentCode5 ,
+                      RiskAssessmentCodeError5 ,
+                      RiskAssessmentCode6 ,
+                      RiskAssessmentCodeError6 ,
+                      RiskAssessmentCode7 ,
+                      RiskAssessmentCodeError7 ,
+                      RiskAssessmentCode8 ,
+                      RiskAssessmentCodeError8 ,
+                      RiskAssessmentCode9 ,
+                      RiskAssessmentCodeError9 ,
+                      RiskAssessmentCode10 ,
+                      RiskAssessmentCodeError10 ,
+                      HashDiff ,
+                      LoadDate ,
+                      RecordSource
+		          )
+                    SELECT  r.S_RAPS_Response_CCC_RK ,
+                            r.H_RAPS_Response_RK ,
+                            r.RecordID ,
+                            r.SeqNo ,
+                            r.SeqErrorCode ,
+                            r.PatientControlNo ,
+                            r.HicNo ,
+                            r.HicErrorCode ,
+                            r.PatientDOB ,
+                            r.DOBErrorCode ,
+                            r.ProviderType1 ,
+                            r.FromDate1 ,
+                            r.ThruDate1 ,
+                            r.DeleteIND1 ,
+                            r.DiagnosisCode1 ,
+                            r.DiagClstrErrorA1 ,
+                            r.DiagClstrErrorB1 ,
+                            r.ProviderType2 ,
+                            r.FromDate2 ,
+                            r.ThruDate2 ,
+                            r.DeleteIND2 ,
+                            r.DiagnosisCode2 ,
+                            r.DiagClstrErrorA2 ,
+                            r.DiagClstrErrorB2 ,
+                            r.ProviderType3 ,
+                            r.FromDate3 ,
+                            r.ThruDate3 ,
+                            r.DeleteIND3 ,
+                            r.DiagnosisCode3 ,
+                            r.DiagClstrErrorA3 ,
+                            r.DiagClstrErrorB3 ,
+                            r.ProviderType4 ,
+                            r.FromDate4 ,
+                            r.ThruDate4 ,
+                            r.DeleteIND4 ,
+                            r.DiagnosisCode4 ,
+                            r.DiagClstrErrorA4 ,
+                            r.DiagClstrErrorB4 ,
+                            r.ProviderType5 ,
+                            r.FromDate5 ,
+                            r.ThruDate5 ,
+                            r.DeleteIND5 ,
+                            r.DiagnosisCode5 ,
+                            r.DiagClstrErrorA5 ,
+                            r.DiagClstrErrorB5 ,
+                            r.ProviderType6 ,
+                            r.FromDate6 ,
+                            r.ThruDate6 ,
+                            r.DeleteIND6 ,
+                            r.DiagnosisCode6 ,
+                            r.DiagClstrErrorA6 ,
+                            r.DiagClstrErrorB6 ,
+                            r.ProviderType7 ,
+                            r.FromDate7 ,
+                            r.ThruDate7 ,
+                            r.DeleteIND7 ,
+                            r.DiagnosisCode7 ,
+                            r.DiagClstrErrorA7 ,
+                            r.DiagClstrErrorB7 ,
+                            r.ProviderType8 ,
+                            r.FromDate8 ,
+                            r.ThruDate8 ,
+                            r.DeleteIND8 ,
+                            r.DiagnosisCode8 ,
+                            r.DiagClstrErrorA8 ,
+                            r.DiagClstrErrorB8 ,
+                            r.ProviderType9 ,
+                            r.FromDate9 ,
+                            r.ThruDate9 ,
+                            r.DeleteIND9 ,
+                            r.DiagnosisCode9 ,
+                            r.DiagClstrErrorA9 ,
+                            r.DiagClstrErrorB9 ,
+                            r.ProviderType10 ,
+                            r.FromDate10 ,
+                            r.ThruDate10 ,
+                            r.DeleteIND10 ,
+                            r.DiagnosisCode10 ,
+                            r.DiagClstrErrorA10 ,
+                            r.DiagClstrErrorB10 ,
+                            r.CorrectedHicNo ,
+                            r.RiskAssessmentCode1 ,
+                            r.RiskAssessmentCodeError1 ,
+                            r.RiskAssessmentCode2 ,
+                            r.RiskAssessmentCodeError2 ,
+                            r.RiskAssessmentCode3 ,
+                            r.RiskAssessmentCodeError3 ,
+                            r.RiskAssessmentCode4 ,
+                            r.RiskAssessmentCodeError4 ,
+                            r.RiskAssessmentCode5 ,
+                            r.RiskAssessmentCodeError5 ,
+                            r.RiskAssessmentCode6 ,
+                            r.RiskAssessmentCodeError6 ,
+                            r.RiskAssessmentCode7 ,
+                            r.RiskAssessmentCodeError7 ,
+                            r.RiskAssessmentCode8 ,
+                            r.RiskAssessmentCodeError8 ,
+                            r.RiskAssessmentCode9 ,
+                            r.RiskAssessmentCodeError9 ,
+                            r.RiskAssessmentCode10 ,
+                            r.RiskAssessmentCodeError10 ,
+                            r.HashDiff ,
+                            r.LoadDate ,
+                            r.RecordSource
+                    FROM    CHSStaging.raps.RAPS_RESPONSE_CCC r
+                            LEFT JOIN dbo.S_RAPS_Response_CCC c ON c.H_RAPS_Response_RK = r.H_RAPS_Response_RK
+                                                              AND c.RecordEndDate IS NULL
+                                                              AND c.HashDiff = r.HashDiff
+                    WHERE   c.S_RAPS_Response_CCC_RK IS NULL; 
+		  
+		  
+		  --RECORD END DATE CLEANUP
+            UPDATE  dbo.S_RAPS_Response_CCC
+            SET     RecordEndDate = ( SELECT    DATEADD(ss, -1,
+                                                        MIN(z.LoadDate))
+                                      FROM      dbo.S_RAPS_Response_CCC AS z
+                                      WHERE     z.H_RAPS_Response_RK = a.H_RAPS_Response_RK
+                                                AND z.LoadDate > a.LoadDate
+                                    )
+            FROM    dbo.S_RAPS_Response_CCC a
+            WHERE   a.RecordEndDate IS NULL;
+		  
+		  
+		  --LOAD RAPS YYY
+            INSERT  INTO dbo.S_RAPS_Response_YYY
+                    ( S_RAPS_Response_YYY_RK ,
+                      H_RAPS_Response_RK ,
+                      RecordID ,
+                      SeqNo ,
+                      PlanNo ,
+                      CCCRecordTotal ,
+                      HashDiff ,
+                      LoadDate ,
+                      RecordSource 
+		          )
+                    SELECT  r.S_RAPS_Response_YYY_RK ,
+                            r.H_RAPS_Response_RK ,
+                            r.RecordID ,
+                            r.SeqNo ,
+                            r.PlanNo ,
+                            r.CCCRecordTotal ,
+                            r.HashDiff ,
+                            r.LoadDate ,
+                            r.RecordSource
+                    FROM    CHSStaging.raps.vwRAPS_RESPONSE_YYY r
+                            LEFT JOIN dbo.S_RAPS_Response_YYY y ON y.H_RAPS_Response_RK = r.H_RAPS_Response_RK
+                                                              AND y.HashDiff = r.HashDiff
+                                                              AND y.RecordEndDate IS NULL
+                    WHERE   y.S_RAPS_Response_YYY_RK IS NULL; 
+		  
+		  --RECORD END DATE CLEANUP
+            UPDATE  dbo.S_RAPS_Response_YYY
+            SET     RecordEndDate = ( SELECT    DATEADD(ss, -1,
+                                                        MIN(z.LoadDate))
+                                      FROM      dbo.S_RAPS_Response_YYY AS z
+                                      WHERE     z.H_RAPS_Response_RK = a.H_RAPS_Response_RK
+                                                AND z.LoadDate > a.LoadDate
+                                    )
+            FROM    dbo.S_RAPS_Response_YYY a
+            WHERE   a.RecordEndDate IS NULL;
+		  
+		  
+		  --LOAD RAPS ZZZ
+            INSERT  INTO dbo.S_RAPS_Response_ZZZ
+                    ( S_RAPS_Response_ZZZ_RK ,
+                      H_RAPS_Response_RK ,
+                      RecordID ,
+                      SubmitterID ,
+                      FileID ,
+                      BBBRecordTotal ,
+                      HashDiff ,
+                      LoadDate ,
+                      RecordSource 
+		          )
+                    SELECT  r.S_RAPS_Response_ZZZ_RK ,
+                            r.H_RAPS_Response_RK ,
+                            r.RecordID ,
+                            r.SubmitterID ,
+                            r.FileID ,
+                            r.BBBRecordTotal ,
+                            r.HashDiff ,
+                            r.LoadDate ,
+                            r.RecordSource
+                    FROM    CHSStaging.raps.vwRAPS_RESPONSE_ZZZ r
+                            LEFT JOIN dbo.S_RAPS_Response_ZZZ z ON z.H_RAPS_Response_RK = r.H_RAPS_Response_RK
+                                                              AND z.HashDiff = r.HashDiff
+                                                              AND z.RecordEndDate IS NULL
+                    WHERE   z.S_RAPS_Response_ZZZ_RK IS NULL; 
+
+
+		  --RECORD END DATE CLEANUP
+            UPDATE  dbo.S_RAPS_Response_ZZZ
+            SET     RecordEndDate = ( SELECT    DATEADD(ss, -1,
+                                                        MIN(z.LoadDate))
+                                      FROM      dbo.S_RAPS_Response_ZZZ AS z
+                                      WHERE     z.H_RAPS_Response_RK = a.H_RAPS_Response_RK
+                                                AND z.LoadDate > a.LoadDate
+                                    )
+            FROM    dbo.S_RAPS_Response_ZZZ a
+            WHERE   a.RecordEndDate IS NULL;
+
+		  
+		  --LOAD MemberHICN satellite
+            INSERT  INTO dbo.S_MemberHICN
+                    ( S_MemberHICN_RK ,
+                      LoadDate ,
+                      H_Member_RK ,
+                      HICNumber ,
+                      HashDiff ,
+                      RecordSource 
+		          )
+                    SELECT DISTINCT
+                            r.S_MemberHICN_RK ,
+                            r.LoadDate ,
+                            r.H_Member_RK ,
+                            r.HicNo ,
+                            r.S_MemberHICN_HashDiff ,
+                            r.RecordSource
+                    FROM    CHSStaging.raps.RAPS_RESPONSE_CCC r
+                            LEFT JOIN dbo.S_MemberHICN s ON s.H_Member_RK = r.H_Member_RK
+                                                            AND s.RecordEndDate IS NULL
+                                                            AND r.S_MemberHICN_HashDiff = s.HashDiff
+                    WHERE   s.S_MemberHICN_RK IS NULL; 
+
+
+		  --RECORD END DATE CLEANUP
+            UPDATE  dbo.S_MemberHICN
+            SET     RecordEndDate = ( SELECT    DATEADD(ss, -1,
+                                                        MIN(z.LoadDate))
+                                      FROM      dbo.S_MemberHICN AS z
+                                      WHERE     z.H_Member_RK = a.H_Member_RK
+                                                AND z.LoadDate > a.LoadDate
+                                    )
+            FROM    dbo.S_MemberHICN a
+            WHERE   a.RecordEndDate IS NULL;
+
+
+
+        END TRY
+        BEGIN CATCH
+             --IF @@TRANCOUNT > 0
+             --    ROLLBACK TRANSACTION;
+            THROW;
+        END CATCH;
+    END;
+
+
+GO
diff --git a/Stored Procedures/dw.spGetRAPSMember.sql b/Stored Procedures/dw.spGetRAPSMember.sql
new file mode 100644
index 0000000..7f9b0c1
--- /dev/null
+++ b/Stored Procedures/dw.spGetRAPSMember.sql
@@ -0,0 +1,23 @@
+SET QUOTED_IDENTIFIER ON
+GO
+SET ANSI_NULLS ON
+GO
+---- =============================================
+---- Author:		Travis Parker
+---- Create date:	06/11/2016
+---- Description:	Gets the latest RAPS Response member data for loading into the DW
+---- Usage:			
+----		  EXECUTE dw.spGetRAPSMember '06/10/2016'
+---- =============================================
+CREATE PROC [dw].[spGetRAPSMember] @LastLoadTime DATETIME
+AS
+    SET NOCOUNT ON; 
+
+    SELECT DISTINCT
+            h.Member_BK AS CentauriMemberID 
+    FROM    dbo.H_Member h
+            INNER JOIN dbo.L_MemberRAPSResponse l ON l.H_Member_RK = h.H_Member_RK
+    WHERE   l.RecordEndDate IS NULL AND h.LoadDate > @LastLoadTime;
+
+
+GO
diff --git a/Stored Procedures/dw.spGetRAPSMemberClient.sql b/Stored Procedures/dw.spGetRAPSMemberClient.sql
new file mode 100644
index 0000000..6d14a3b
--- /dev/null
+++ b/Stored Procedures/dw.spGetRAPSMemberClient.sql
@@ -0,0 +1,27 @@
+SET QUOTED_IDENTIFIER ON
+GO
+SET ANSI_NULLS ON
+GO
+---- =============================================
+---- Author:		Travis Parker
+---- Create date:	06/11/2016
+---- Description:	Gets the latest RAPS Response MemberClient data for loading into the DW
+---- Usage:			
+----		  EXECUTE dw.spGetRAPSMemberClient '06/10/2016'
+---- =============================================
+CREATE PROC [dw].[spGetRAPSMemberClient]
+    @LastLoadTime DATETIME
+AS
+    SET NOCOUNT ON; 
+
+    SELECT  DISTINCT
+            h.Member_BK AS CentauriMemberID ,
+            c.Client_BK AS CentauriClientID ,
+            h.ClientMemberID
+    FROM    dbo.H_Member h
+            INNER JOIN dbo.L_MemberRAPSResponse l ON l.H_Member_RK = h.H_Member_RK
+            CROSS JOIN dbo.H_Client c
+    WHERE   l.RecordEndDate IS NULL
+            AND l.LoadDate > @LastLoadTime;
+
+GO
diff --git a/Stored Procedures/dw.spGetRAPSMemberHICN.sql b/Stored Procedures/dw.spGetRAPSMemberHICN.sql
new file mode 100644
index 0000000..28bc450
--- /dev/null
+++ b/Stored Procedures/dw.spGetRAPSMemberHICN.sql
@@ -0,0 +1,24 @@
+SET QUOTED_IDENTIFIER ON
+GO
+SET ANSI_NULLS ON
+GO
+---- =============================================
+---- Author:		Travis Parker
+---- Create date:	06/11/2016
+---- Description:	Gets the latest RAPS Response member HICN data for loading into the DW
+---- Usage:			
+----		  EXECUTE dw.spGetRAPSMemberHICN '06/10/2016'
+---- =============================================
+CREATE PROC [dw].[spGetRAPSMemberHICN] @LastLoadTime DATETIME
+AS
+    SET NOCOUNT ON; 
+
+    SELECT DISTINCT
+            h.Member_BK AS CentauriMemberID ,
+		  s.HICNumber
+    FROM    dbo.H_Member h
+            INNER JOIN dbo.L_MemberRAPSResponse l ON l.H_Member_RK = h.H_Member_RK
+		  INNER JOIN dbo.S_MemberHICN s ON s.H_Member_RK = h.H_Member_RK
+    WHERE   s.RecordEndDate IS NULL AND s.LoadDate > @LastLoadTime;
+
+GO
diff --git a/Stored Procedures/dw.spGetRAPSResponse.sql b/Stored Procedures/dw.spGetRAPSResponse.sql
new file mode 100644
index 0000000..44fc5fc
--- /dev/null
+++ b/Stored Procedures/dw.spGetRAPSResponse.sql
@@ -0,0 +1,80 @@
+SET QUOTED_IDENTIFIER ON
+GO
+SET ANSI_NULLS ON
+GO
+---- =============================================
+---- Author:		Travis Parker
+---- Create date:	06/11/2016
+---- Description:	Gets the latest RAPS Response data for loading into the DW
+---- Usage:			
+----		  EXECUTE dw.spGetRAPSResponse '06/10/2016'
+---- =============================================
+CREATE PROC [dw].[spGetRAPSResponse]
+    @LastLoadTime DATETIME
+AS
+    SET NOCOUNT ON; 
+
+    SELECT  cl.Client_BK AS CentauriClientID ,
+            m.Member_BK AS CentauriMemberID ,
+            a.FileID ,
+            c.SeqNo ,
+            c.HicNo ,
+            RTRIM(c.PatientControlNo) AS 'PatientControlNo' ,
+            CONVERT(VARCHAR(50),dbo.ufn_parsefind(c.PatientControlNo, '-', 2)) AS ClaimNumber ,
+            c.PatientDOB ,
+            c.DOBErrorCode ,
+            c.HicErrorCode ,
+            b.PlanNo ,
+            pvt.FromDate ,
+            pvt.ThruDate ,
+            pvt.ProviderType ,
+            pvt.DXCode ,
+            pvt.ErrorA ,
+            pvt.ErrorB ,
+            pvt.RiskAssessmentCode ,
+            pvt.RiskAssessmentCodeError ,
+            pvt.ClusterGrouping ,
+            a.TransactionDate ,
+            a.FileDiagType ,
+            CASE WHEN COALESCE(REPLACE(c.DOBErrorCode, '500', ''),
+                               REPLACE(c.HicErrorCode, '500', ''),
+                               REPLACE(pvt.ErrorA, '500', ''),
+                               REPLACE(pvt.ErrorB, '500', ''), '') <> ''
+                 THEN 0
+                 ELSE 1
+            END AS Accepted,
+		  c.RecordSource AS FileName
+    FROM    dbo.S_RAPS_Response_CCC c
+            INNER JOIN dbo.S_RAPS_Response_AAA a ON a.H_RAPS_Response_RK = c.H_RAPS_Response_RK
+            INNER JOIN dbo.S_RAPS_Response_BBB b ON b.H_RAPS_Response_RK = a.H_RAPS_Response_RK
+            INNER JOIN dbo.L_MemberRAPSResponse l ON l.H_RAPS_Response_RK = a.H_RAPS_Response_RK
+            INNER JOIN dbo.H_Member m ON m.H_Member_RK = l.H_Member_RK
+            CROSS JOIN dbo.H_Client cl
+            CROSS APPLY ( VALUES
+            ( c.FromDate1, c.ThruDate1, c.ProviderType1, c.DiagnosisCode1, c.DiagClstrErrorA1, c.DiagClstrErrorB1, c.RiskAssessmentCode1, c.RiskAssessmentCodeError1, 1)
+					,
+            ( c.FromDate2, c.ThruDate2, c.ProviderType2, c.DiagnosisCode2, c.DiagClstrErrorA2, c.DiagClstrErrorB2, c.RiskAssessmentCode2, c.RiskAssessmentCodeError2, 2)
+					,
+            ( c.FromDate3, c.ThruDate3, c.ProviderType3, c.DiagnosisCode3, c.DiagClstrErrorA3, c.DiagClstrErrorB3, c.RiskAssessmentCode3, c.RiskAssessmentCodeError3, 3)
+					,
+            ( c.FromDate4, c.ThruDate4, c.ProviderType4, c.DiagnosisCode4, c.DiagClstrErrorA4, c.DiagClstrErrorB4, c.RiskAssessmentCode4, c.RiskAssessmentCodeError4, 4)
+					,
+            ( c.FromDate5, c.ThruDate5, c.ProviderType5, c.DiagnosisCode5, c.DiagClstrErrorA5, c.DiagClstrErrorB5, c.RiskAssessmentCode5, c.RiskAssessmentCodeError5, 5)
+					,
+            ( c.FromDate6, c.ThruDate6, c.ProviderType6, c.DiagnosisCode6, c.DiagClstrErrorA6, c.DiagClstrErrorB6, c.RiskAssessmentCode6, c.RiskAssessmentCodeError6, 6)
+					,
+            ( c.FromDate7, c.ThruDate7, c.ProviderType7, c.DiagnosisCode7, c.DiagClstrErrorA7, c.DiagClstrErrorB7, c.RiskAssessmentCode7, c.RiskAssessmentCodeError7, 7)
+					,
+            ( c.FromDate8, c.ThruDate8, c.ProviderType8, c.DiagnosisCode8, c.DiagClstrErrorA8, c.DiagClstrErrorB8, c.RiskAssessmentCode8, c.RiskAssessmentCodeError8, 8)
+					,
+            ( c.FromDate9, c.ThruDate9, c.ProviderType9, c.DiagnosisCode9, c.DiagClstrErrorA9, c.DiagClstrErrorB9, c.RiskAssessmentCode9, c.RiskAssessmentCodeError9, 9)
+					,
+            ( c.FromDate10, c.ThruDate10, c.ProviderType10, c.DiagnosisCode10, c.DiagClstrErrorA10, c.DiagClstrErrorB10, c.RiskAssessmentCode10, c.RiskAssessmentCodeError10, 10) ) pvt ( FromDate, ThruDate, ProviderType, DXCode, ErrorA, ErrorB, RiskAssessmentCode, RiskAssessmentCodeError, ClusterGrouping )
+    WHERE   ISNULL(pvt.FromDate, '') <> ''
+            AND ISNULL(pvt.ThruDate, '') <> ''
+            AND ISNULL(pvt.ProviderType, '') <> ''
+            AND ISNULL(pvt.DXCode, '') <> ''
+            AND c.LoadDate > @LastLoadTime;
+
+
+GO
diff --git a/Tables/dbo.H_Client.sql b/Tables/dbo.H_Client.sql
new file mode 100644
index 0000000..7b03190
--- /dev/null
+++ b/Tables/dbo.H_Client.sql
@@ -0,0 +1,11 @@
+CREATE TABLE [dbo].[H_Client]
+(
+[H_Client_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[Client_BK] [int] NOT NULL,
+[ClientName] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RecordSource] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[LoadDate] [datetime] NULL CONSTRAINT [DF_H_Client_LoadDate] DEFAULT (getdate())
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[H_Client] ADD CONSTRAINT [PK_H_Client] PRIMARY KEY CLUSTERED  ([H_Client_RK]) ON [PRIMARY]
+GO
diff --git a/Tables/dbo.H_Member.sql b/Tables/dbo.H_Member.sql
new file mode 100644
index 0000000..5139730
--- /dev/null
+++ b/Tables/dbo.H_Member.sql
@@ -0,0 +1,11 @@
+CREATE TABLE [dbo].[H_Member]
+(
+[H_Member_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[Member_BK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ClientMemberID] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[LoadDate] [datetime] NULL CONSTRAINT [DF_H_Member_LoadDate] DEFAULT (getdate()),
+[RecordSource] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[H_Member] ADD CONSTRAINT [PK_H_Member] PRIMARY KEY CLUSTERED  ([H_Member_RK]) ON [PRIMARY]
+GO
diff --git a/Tables/dbo.H_RAPS_Response.sql b/Tables/dbo.H_RAPS_Response.sql
new file mode 100644
index 0000000..7ca88a1
--- /dev/null
+++ b/Tables/dbo.H_RAPS_Response.sql
@@ -0,0 +1,10 @@
+CREATE TABLE [dbo].[H_RAPS_Response]
+(
+[H_RAPS_Response_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[RAPS_Response_BK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[LoadDate] [datetime] NOT NULL CONSTRAINT [DF_H_RAPS_Response_LoadDate] DEFAULT (getdate()),
+[RecordSource] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[H_RAPS_Response] ADD CONSTRAINT [PK_H_RAPS_Response] PRIMARY KEY CLUSTERED  ([H_RAPS_Response_RK]) ON [PRIMARY]
+GO
diff --git a/Tables/dbo.L_MemberRAPSResponse.sql b/Tables/dbo.L_MemberRAPSResponse.sql
new file mode 100644
index 0000000..3315d59
--- /dev/null
+++ b/Tables/dbo.L_MemberRAPSResponse.sql
@@ -0,0 +1,12 @@
+CREATE TABLE [dbo].[L_MemberRAPSResponse]
+(
+[L_MemberRAPSResponse_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[H_RAPS_Response_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[H_Member_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[LoadDate] [datetime] NOT NULL CONSTRAINT [DF_L_MemberRAPSResponse_LoadDate] DEFAULT (getdate()),
+[RecordSource] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[RecordEndDate] [datetime] NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[L_MemberRAPSResponse] ADD CONSTRAINT [PK_L_MemberRAPSResponse] PRIMARY KEY CLUSTERED  ([L_MemberRAPSResponse_RK]) ON [PRIMARY]
+GO
diff --git a/Tables/dbo.S_MemberDemo.sql b/Tables/dbo.S_MemberDemo.sql
new file mode 100644
index 0000000..816ac72
--- /dev/null
+++ b/Tables/dbo.S_MemberDemo.sql
@@ -0,0 +1,22 @@
+CREATE TABLE [dbo].[S_MemberDemo]
+(
+[S_MemberDemo_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[LoadDate] [datetime] NOT NULL CONSTRAINT [DF_S_MemberDemo_LoadDate] DEFAULT (getdate()),
+[H_Member_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[FirstName] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[LastName] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[Gender] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DOB] [datetime] NULL,
+[HashDiff] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RecordSource] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RecordEndDate] [datetime] NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[S_MemberDemo] ADD CONSTRAINT [PK_S_MemberDemo_1] PRIMARY KEY CLUSTERED  ([S_MemberDemo_RK]) ON [PRIMARY]
+GO
+CREATE NONCLUSTERED INDEX [NonClusteredIndex-20160921-131718] ON [dbo].[S_MemberDemo] ([HashDiff]) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[S_MemberDemo] ADD CONSTRAINT [FK_H_Member_RK4] FOREIGN KEY ([H_Member_RK]) REFERENCES [dbo].[H_Member] ([H_Member_RK]) ON DELETE CASCADE ON UPDATE CASCADE
+GO
+ALTER TABLE [dbo].[S_MemberDemo] ADD CONSTRAINT [FK_S_MemberDemo_H_Member] FOREIGN KEY ([H_Member_RK]) REFERENCES [dbo].[H_Member] ([H_Member_RK])
+GO
diff --git a/Tables/dbo.S_MemberHICN.sql b/Tables/dbo.S_MemberHICN.sql
new file mode 100644
index 0000000..89a6291
--- /dev/null
+++ b/Tables/dbo.S_MemberHICN.sql
@@ -0,0 +1,17 @@
+CREATE TABLE [dbo].[S_MemberHICN]
+(
+[S_MemberHICN_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[LoadDate] [datetime] NOT NULL,
+[H_Member_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[HICNumber] [varchar] (20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[HashDiff] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RecordSource] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RecordEndDate] [datetime] NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[S_MemberHICN] ADD CONSTRAINT [PK_S_MemberHICN] PRIMARY KEY CLUSTERED  ([S_MemberHICN_RK]) ON [PRIMARY]
+GO
+CREATE NONCLUSTERED INDEX [NonClusteredIndex-20161003-080959] ON [dbo].[S_MemberHICN] ([HashDiff]) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[S_MemberHICN] ADD CONSTRAINT [FK_S_MemberHICN_H_Member] FOREIGN KEY ([H_Member_RK]) REFERENCES [dbo].[H_Member] ([H_Member_RK])
+GO
diff --git a/Tables/dbo.S_RAPS_Response_AAA.sql b/Tables/dbo.S_RAPS_Response_AAA.sql
new file mode 100644
index 0000000..c8fd8e0
--- /dev/null
+++ b/Tables/dbo.S_RAPS_Response_AAA.sql
@@ -0,0 +1,20 @@
+CREATE TABLE [dbo].[S_RAPS_Response_AAA]
+(
+[S_RAPS_Response_AAA_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[H_RAPS_Response_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[RecordID] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[SubmitterID] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[FileID] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[TransactionDate] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ProdTestIND] [varchar] (20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[FileDiagType] [varchar] (20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[HashDiff] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[LoadDate] [datetime] NOT NULL CONSTRAINT [DF_S_RAPS_Response_AAA_LoadDate] DEFAULT (getdate()),
+[RecordSource] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[RecordEndDate] [datetime] NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[S_RAPS_Response_AAA] ADD CONSTRAINT [PK_S_RAPS_Response_AAA] PRIMARY KEY CLUSTERED  ([S_RAPS_Response_AAA_RK]) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[S_RAPS_Response_AAA] ADD CONSTRAINT [FK_S_RAPS_Response_AAA_H_RAPS_Response] FOREIGN KEY ([H_RAPS_Response_RK]) REFERENCES [dbo].[H_RAPS_Response] ([H_RAPS_Response_RK])
+GO
diff --git a/Tables/dbo.S_RAPS_Response_BBB.sql b/Tables/dbo.S_RAPS_Response_BBB.sql
new file mode 100644
index 0000000..b049285
--- /dev/null
+++ b/Tables/dbo.S_RAPS_Response_BBB.sql
@@ -0,0 +1,21 @@
+CREATE TABLE [dbo].[S_RAPS_Response_BBB]
+(
+[S_RAPS_Response_BBB_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[H_RAPS_Response_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[RecordID] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[SeqNo] [varchar] (20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[PlanNo] [varchar] (20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[OverpaymentID] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[OverpaymentIDErrorCode] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[PaymentYear] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[PaymentYearErrorCode] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[HashDiff] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[LoadDate] [datetime] NOT NULL CONSTRAINT [DF_S_RAPS_Response_BBB_LoadDate] DEFAULT (getdate()),
+[RecordSource] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[RecordEndDate] [datetime] NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[S_RAPS_Response_BBB] ADD CONSTRAINT [PK_S_RAPS_Response_BBB] PRIMARY KEY CLUSTERED  ([S_RAPS_Response_BBB_RK]) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[S_RAPS_Response_BBB] ADD CONSTRAINT [FK_S_RAPS_Response_BBB_H_RAPS_Response] FOREIGN KEY ([H_RAPS_Response_RK]) REFERENCES [dbo].[H_RAPS_Response] ([H_RAPS_Response_RK])
+GO
diff --git a/Tables/dbo.S_RAPS_Response_CCC.sql b/Tables/dbo.S_RAPS_Response_CCC.sql
new file mode 100644
index 0000000..e8bb288
--- /dev/null
+++ b/Tables/dbo.S_RAPS_Response_CCC.sql
@@ -0,0 +1,113 @@
+CREATE TABLE [dbo].[S_RAPS_Response_CCC]
+(
+[S_RAPS_Response_CCC_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[H_RAPS_Response_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[RecordID] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[SeqNo] [varchar] (20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[SeqErrorCode] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[PatientControlNo] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[HicNo] [varchar] (20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[HicErrorCode] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[PatientDOB] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DOBErrorCode] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ProviderType1] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[FromDate1] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ThruDate1] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DeleteIND1] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagnosisCode1] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorA1] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorB1] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ProviderType2] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[FromDate2] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ThruDate2] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DeleteIND2] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagnosisCode2] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorA2] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorB2] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ProviderType3] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[FromDate3] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ThruDate3] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DeleteIND3] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagnosisCode3] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorA3] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorB3] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ProviderType4] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[FromDate4] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ThruDate4] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DeleteIND4] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagnosisCode4] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorA4] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorB4] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ProviderType5] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[FromDate5] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ThruDate5] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DeleteIND5] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagnosisCode5] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorA5] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorB5] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ProviderType6] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[FromDate6] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ThruDate6] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DeleteIND6] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagnosisCode6] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorA6] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorB6] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ProviderType7] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[FromDate7] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ThruDate7] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DeleteIND7] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagnosisCode7] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorA7] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorB7] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ProviderType8] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[FromDate8] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ThruDate8] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DeleteIND8] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagnosisCode8] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorA8] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorB8] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ProviderType9] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[FromDate9] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ThruDate9] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DeleteIND9] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagnosisCode9] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorA9] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorB9] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ProviderType10] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[FromDate10] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[ThruDate10] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DeleteIND10] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagnosisCode10] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorA10] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[DiagClstrErrorB10] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[CorrectedHicNo] [varchar] (20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCode1] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCodeError1] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCode2] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCodeError2] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCode3] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCodeError3] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCode4] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCodeError4] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCode5] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCodeError5] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCode6] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCodeError6] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCode7] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCodeError7] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCode8] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCodeError8] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCode9] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCodeError9] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCode10] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[RiskAssessmentCodeError10] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[HashDiff] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[LoadDate] [datetime] NOT NULL CONSTRAINT [DF_S_RAPS_Response_CCC_LoadDate] DEFAULT (getdate()),
+[RecordSource] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[RecordEndDate] [datetime] NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[S_RAPS_Response_CCC] ADD CONSTRAINT [PK_S_RAPS_Response_CCC] PRIMARY KEY CLUSTERED  ([S_RAPS_Response_CCC_RK]) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[S_RAPS_Response_CCC] ADD CONSTRAINT [FK_S_RAPS_Response_CCC_H_RAPS_Response] FOREIGN KEY ([H_RAPS_Response_RK]) REFERENCES [dbo].[H_RAPS_Response] ([H_RAPS_Response_RK])
+GO
diff --git a/Tables/dbo.S_RAPS_Response_YYY.sql b/Tables/dbo.S_RAPS_Response_YYY.sql
new file mode 100644
index 0000000..fd6a026
--- /dev/null
+++ b/Tables/dbo.S_RAPS_Response_YYY.sql
@@ -0,0 +1,18 @@
+CREATE TABLE [dbo].[S_RAPS_Response_YYY]
+(
+[S_RAPS_Response_YYY_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[H_RAPS_Response_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[RecordID] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[SeqNo] [varchar] (20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[PlanNo] [varchar] (20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[CCCRecordTotal] [varchar] (20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[HashDiff] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[LoadDate] [datetime] NOT NULL CONSTRAINT [DF_S_RAPS_Response_YYY_LoadDate] DEFAULT (getdate()),
+[RecordSource] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[RecordEndDate] [datetime] NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[S_RAPS_Response_YYY] ADD CONSTRAINT [PK_S_RAPS_Response_YYY] PRIMARY KEY CLUSTERED  ([S_RAPS_Response_YYY_RK]) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[S_RAPS_Response_YYY] ADD CONSTRAINT [FK_S_RAPS_Response_YYY_H_RAPS_Response] FOREIGN KEY ([H_RAPS_Response_RK]) REFERENCES [dbo].[H_RAPS_Response] ([H_RAPS_Response_RK])
+GO
diff --git a/Tables/dbo.S_RAPS_Response_ZZZ.sql b/Tables/dbo.S_RAPS_Response_ZZZ.sql
new file mode 100644
index 0000000..abf7da5
--- /dev/null
+++ b/Tables/dbo.S_RAPS_Response_ZZZ.sql
@@ -0,0 +1,18 @@
+CREATE TABLE [dbo].[S_RAPS_Response_ZZZ]
+(
+[S_RAPS_Response_ZZZ_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[H_RAPS_Response_RK] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[RecordID] [varchar] (10) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[SubmitterID] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[FileID] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[BBBRecordTotal] [varchar] (20) COLLATE SQL_Latin1_General_CP1_CI_AS NULL,
+[HashDiff] [varchar] (50) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[LoadDate] [datetime] NOT NULL CONSTRAINT [DF_S_RAPS_Response_ZZZ_LoadDate] DEFAULT (getdate()),
+[RecordSource] [varchar] (100) COLLATE SQL_Latin1_General_CP1_CI_AS NOT NULL,
+[RecordEndDate] [datetime] NULL
+) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[S_RAPS_Response_ZZZ] ADD CONSTRAINT [PK_S_RAPS_Response_ZZZ] PRIMARY KEY CLUSTERED  ([S_RAPS_Response_ZZZ_RK]) ON [PRIMARY]
+GO
+ALTER TABLE [dbo].[S_RAPS_Response_ZZZ] ADD CONSTRAINT [FK_S_RAPS_Response_ZZZ_H_RAPS_Response] FOREIGN KEY ([H_RAPS_Response_RK]) REFERENCES [dbo].[H_RAPS_Response] ([H_RAPS_Response_RK])
+GO
